pkg/skaffold/deploy/helm/util.go

```diff
@@ -242,3 +220,8 @@ func (h *Deployer) exec(ctx context.Context, out io.Writer, useSecrets bool, env
 
 	return util.RunCmd(cmd)
 }
+
+func version() ([]byte, error) {
```

Comment:
Move this up to be closer to `binVer()`?  And why don't we just inline it?

Reply:
done, inlined it.